portfolio/src/main/webapp/games.html

```diff
@@ -32,11 +33,22 @@
   <!-- Games Section -->
   <a id="games">
     <div class="game-section">
+      <div class="game-section-header">
+          <h1>Games</h1>
+          <p>Play some of the games I've made, leave comments, and challenge yourself
+               and other players by making it to the top of the leaderboards!
```

Comment:
This is interesting.  Not sure if I missed, but how does it keep track of high scores?  Does it persist on page refresh?

Reply:
I've saved high scores to the datastore, and thus far w/o being in a live server they seem to persist